discord/utils.py

```diff
@@ -448,14 +448,11 @@ def get(iterable: Iterable[T], **attrs: Any) -> Optional[T]:
             return elem
     return None
 
-async def get_or_fetch(obj, attr: str, id: int, *, default: Any = None):
+async def get_or_fetch(obj, attr: str, id: int):
```

Comment:
Why did you remove the default param? I think you should merge from slash once to sync the changes.

Reply:
Already reverted, accidentally overwritten it.